ding_availability.module

```diff
@@ -184,7 +185,25 @@ function ding_availability_holdings($provider_ids) {
       );
 
       $rows = array();
+      $copies = 0;
+      $closest_loan = NULL;
+      $in_acquisition = 0;
       foreach ($item['holdings'] as $holding) {
+        // This item is considered to be waited for.
+        // Next, the number of holdings is compared to reserved ones.
+        // Also keep the next loan date.
+        if (empty($item['available']) && !empty($holding['available_from'])) {
+          $next_loan_time = strtotime($holding['available_from']);
+          $copies++;
+          if (empty($closest_loan) || $closest_loan < $next_loan_time) {
+            $closest_loan = $next_loan_time;
+          }
+        }
+
+        if ($holding['status'] == 'inAcquisition') {
+          $in_acquisition++;
+        }
+
```

Comment:
Is this change intentionally part of this pull request?

Extracting `closest_loan` and `in_acquisition` does not seem to be related to [issue 618](http://platform.dandigbib.org/issues/618). I would also think that supporting this would require changes to the provider implementations.